CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing to HTTPS Everywhere
```

Comment:
Since this is only about contributions of rulesets, perhaps that should be reflected in the title/naming?